concepticondata/conceptlists/Winter-2022-98.tsv

```diff
@@ -1,99 +1,99 @@
+Winter-2022-98-57	57	estuary	2286	ESTUARY	"[{""NAME"": ""mouth"", ""ID"": ""Winter-2022-98-32"", ""OvertMarking"": 5}]"	"[{""NAME"": ""mouth"", ""ID"": ""Winter-2022-98-32"", ""OvertMarking"": 0}]"	"[{""NAME"": ""mouth"", ""ID"": ""Winter-2022-98-32"", ""Polysemy"": 5}]"
 Winter-2022-98-58	58	tongue	1205	TONGUE	"[{""NAME"": ""flame"", ""ID"": ""Winter-2022-98-59"", ""OvertMarking"": 0}]"	"[{""NAME"": ""flame"", ""ID"": ""Winter-2022-98-59"", ""OvertMarking"": 24}]"	"[{""NAME"": ""flame"", ""ID"": ""Winter-2022-98-59"", ""Polysemy"": 2}]"
-Winter-2022-98-59	59	flame	889	FLAME	"[{""NAME"": ""tongue"", ""ID"": ""Winter-2022-98-58"", ""OvertMarking"": 24}, {""NAME"": ""embers"", ""ID"": ""Winter-2022-98-17"", ""OvertMarking"": 2}]"	"[{""NAME"": ""tongue"", ""ID"": ""Winter-2022-98-58"", ""OvertMarking"": 0}, {""NAME"": ""embers"", ""ID"": ""Winter-2022-98-17"", ""OvertMarking"": 0}]"	"[{""NAME"": ""flame"", ""ID"": ""Winter-2022-98-59"", ""Polysemy"": 2}, {""NAME"": ""flame"", ""ID"": ""Winter-2022-98-59"", ""Polysemy"": 6}]"
-Winter-2022-98-60	60	road/street/way	667	ROAD	"[{""NAME"": ""milky way"", ""ID"": ""Winter-2022-98-61"", ""OvertMarking"": 0}]"	"[{""NAME"": ""milky way"", ""ID"": ""Winter-2022-98-61"", ""OvertMarking"": 15}]"	"[{""NAME"": ""milky way"", ""ID"": ""Winter-2022-98-61"", ""Polysemy"": 2}]"
-Winter-2022-98-61	61	milky way	2289	MILKY WAY	"[{""NAME"": ""road/street/way"", ""ID"": ""Winter-2022-98-60"", ""OvertMarking"": 15}, {""NAME"": ""river/stream"", ""ID"": ""Winter-2022-98-28"", ""OvertMarking"": 0}]"	"[{""NAME"": ""road/street/way"", ""ID"": ""Winter-2022-98-60"", ""OvertMarking"": 0}, {""NAME"": ""river/stream"", ""ID"": ""Winter-2022-98-28"", ""OvertMarking"": 2}]"	"[{""NAME"": ""milky way"", ""ID"": ""Winter-2022-98-61"", ""Polysemy"": 2}, {""NAME"": ""river/stream"", ""ID"": ""Winter-2022-98-28"", ""Polysemy"": 2}]"
+Winter-2022-98-59	59	flame	889	FLAME	"[{""NAME"": ""tongue"", ""ID"": ""Winter-2022-98-58"", ""OvertMarking"": 24}, {""NAME"": ""embers"", ""ID"": ""Winter-2022-98-17"", ""OvertMarking"": 2}]"	"[{""NAME"": ""tongue"", ""ID"": ""Winter-2022-98-58"", ""OvertMarking"": 0}, {""NAME"": ""embers"", ""ID"": ""Winter-2022-98-17"", ""OvertMarking"": 0}]"	"[{""NAME"": ""tongue"", ""ID"": ""Winter-2022-98-58"", ""Polysemy"": 2}, {""NAME"": ""embers"", ""ID"": ""Winter-2022-98-17"", ""Polysemy"": 6}]"
+Winter-2022-98-60	60	road/street/way	2457	PATH OR ROAD	"[{""NAME"": ""milky way"", ""ID"": ""Winter-2022-98-61"", ""OvertMarking"": 0}]"	"[{""NAME"": ""milky way"", ""ID"": ""Winter-2022-98-61"", ""OvertMarking"": 15}]"	"[{""NAME"": ""milky way"", ""ID"": ""Winter-2022-98-61"", ""Polysemy"": 2}]"
```

Comment:
Okay, this had me confused for a second since I could sometimes reproduce this and sometimes not.

`pysem`'s `to_concepticon()` in some cases returns the mapping `[['667', 'ROAD', 'noun', 19]]`, and in some other cases `[['2457', 'PATH OR ROAD', 'noun', 19]]`.